tests/unit/fetch_test.ts

```diff
@@ -1214,7 +1214,7 @@ Deno.test(
       ],
       body: stream.readable,
     });
-    await response.arrayBuffer();
+    await response.body?.cancel();
```

Comment:
Are these `.body?.cancel()` intentional? Why not `.bytes()`?

Reply:
`.body?.cancel()` yields the same effect as `.bytes()` and `.arrayBuffer()`, but with less work.